src/fictionalpancake/turbospork/paint/GraphicsHandler.java

```diff
@@ -32,7 +32,16 @@ public void paint(IPainter g) {
                 PaintStyle style = new PaintStyle();
                 style.color = GameColors.getColorForOwner(node.getOwner());
                 g.drawCircle(style, convertX(node.getX()), convertY(node.getY()), r);
-                drawNodeUnits(g, node);
+                // draw yo moms house
+                if (node.getOwner() == -3) {
+                    Color purple = new Color(0x420666);
+                    for (int circle = GameConstants.HOUSE_NUM_CIRCLES - 1; circle >= 0; circle--) {
+                        style.color = ((circle % 2 == 0) ? Color.BLUE : purple);
```

Comment:
You should probably use two styles and switch between those.  This way doesn't look very good.